app/controllers/reactions_controller.rb

```diff
@@ -79,6 +79,14 @@ def cached_user_positive_reactions(user)
 
   private
 
+  def reaction_user(reaction)
```

Comment:
This could be extracted to the reaction model in a future change, but there is a *similar* method already there and I don't think it's worth digging around too much right now. Maybe that can be a task for after the cycle?